tests/test_bashcomplete.py

```diff
@@ -3,7 +3,18 @@
 import click
 from click._bashcomplete import get_choices
 
-def test_basic():
+
+def test_single_command():
+    @click.command()
+    @click.option('--local-opt')
+    def cli(local_opt):
+        pass
+
+    assert list(get_choices(cli, 'lol', [], '-')) == ['--local-opt']
+    assert list(get_choices(cli, 'lol', [], '')) == []
```

Comment:
The other one: 'def test_long_chain():' does. I just added the test_single_command for completeness. See my comment here: https://github.com/pallets/click/issues/628#issuecomment-241047886